CefSharp.BrowserSubprocess.Core/TypeUtils.cpp

```diff
@@ -94,6 +95,11 @@ namespace CefSharp
         {
             return CefV8Value::CreateDate(TypeUtils::ConvertDateTimeToCefTime(safe_cast<DateTime>(obj)));
         }
+        if (type == JavascriptObjectWrapper::typeid) 
```

Comment:
Is there a better place for this conversion? Something about this just doesn't quite sit right.